pbft/test/pbft_proto_test.cpp

```diff
@@ -56,17 +56,16 @@ namespace bzn
         dmsg->set_allocated_create(create);
         request.set_database_msg(dmsg->SerializeAsString());
         request.set_timestamp(this->now());
+        request.set_sender(this->pbft->get_uuid());
 
         pbft->handle_request(request);
 
         return operation;
     }
 
-    // send a preprepare message to SUT
     void
```

Comment:
Please put the comments back. The ? was because you changed "after preprepare is sent" to "after preprepares is sent"

Reply:
Returned the comments, removed the 's'